components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/BrowserToolbar.kt

```diff
@@ -187,6 +187,12 @@ class BrowserToolbar @JvmOverloads constructor(
             field = value
         }
 
+    override var siteSecure: Toolbar.SiteSecurity = Toolbar.SiteSecurity.INSECURE
+        set(value) {
+            displayToolbar.setSiteSecurity(value)
+            field = value
+        }
```

Comment:
I wonder if we should turn this into `setSiteSecurity` too. Not sure if we want to keep this as a readable property?

Reply:
That's a fair point. I want to say it'll be useful for browser apps that want to read the the toolbar state if they want to style it differently, but they should really be using the observers in that case.

I'll remove the read ability and merge.

Reply:
I thought about it a bit more. It's not that bad to have it visible (for now) and it also makes testing a little easier, so I'm going to leave it as is.